### ISSUE TITLE
Deamon not found

### ISSUE THREAD

**Issue by GalaxyGamer5** [X3n0m0rph59/eruption#57]
So I am the dude from your Reddit post, and I tried the steps you told me to do, but I am sorry to say, the problem is still there.

I don't know why the "bug" occurred. I installed the most recent version of 20.04 Ubuntu about a week ago.
I am pretty sure I made no mistakes in the installation process.

Like mentioned I get this error message: 
![Screenshot from 2021-04-10 21-53-29](https://user-images.githubusercontent.com/82287934/114282959-7d459600-9a47-11eb-9b81-be63bde20813.png)

Then I tried your lines in the Terminal and the result were:
sudo systemctl status eruption.service:
![Screenshot from 2021-04-10 21-47-57](https://user-images.githubusercontent.com/82287934/114282982-9b12fb00-9a47-11eb-852f-2ead729d2abf.png)
and sudo journalctl -u eruption.service -b:
![Screenshot from 2021-04-10 21-48-51](https://user-images.githubusercontent.com/82287934/114283002-b8e06000-9a47-11eb-9168-cbc9d54411bc.png)
![Screenshot from 2021-04-10 21-48-57](https://user-images.githubusercontent.com/82287934/114282930-47081680-9a47-11eb-8aa9-5bff47c45e0e.png)

At the first line the console wouldn't do anything unless I pressed Ctrl+C again.
The second line was in a weird state were, if I pressed enter a new line would write, which would after a while recreate the text that's already written above. 

You asked if my deviced would show LED activity, and they didn't. I mean, yes they did, but only the same way they did before I installed Eruption. (if it does help, I use a Roccat Kone Aimo (the standard Version) and I have no idea if other USB deviced can lead to trubble shooting, but my other two devices are the Hyperx Alloy Elite RGB and the Razer Electra V2 USB, which have both also LEDs in it.)

 * Eruption version:  [0.1.19]
 * Your device(s) :Sudo version 1.8.31
Sudoers policy plugin version 1.8.31
Sudoers file grammar version 46
Sudoers I/O plugin version 1.8.31

**Comment by X3n0m0rph59** [comment]
Hi, thanks for posting this bug report!

It seems like you have hit a bug in version `0.1.19` of Eruption. This build does not fully support running in mouse only mode, without a supported keyboard being attached to the system.

I have attached a pre-release package of Eruption version `0.1.20`. If you want to try it out, please install using the following commands. (Assuming the file has been placed in the `Downloads` directory):
```sh
$ cd Downloads/
$ unzip eruption-git_0.1.20-ubuntu0_amd64.deb.zip
$ sudo apt install ./eruption-git_0.1.20-ubuntu0_amd64.deb
$ sudo systemctl enable --now eruption.service
$ eruptionctl switch profile batique.profile
```

[eruption-git_0.1.20-ubuntu0_amd64.deb.zip](https://github.com/X3n0m0rph59/eruption/files/6291188/eruption-git_0.1.20-ubuntu0_amd64.deb.zip)

**Comment by X3n0m0rph59** [comment]
@GalaxyGamer5

I have released Eruption `0.1.20`, which should fix the bug.
Please re-open this issue or create another one if you encounter further problems!
